internal/handlers/handler_test.go

```diff
@@ -413,6 +417,13 @@ func TestReadOnlyHandlers(t *testing.T) {
 		compareFunc func(t testing.TB, actual, expected any)
 	}
 
+	now := time.Now().UTC()
+	shared.Now = func() time.Time {
```

Comment:
And we never set it back.

I think we should be fine with `testutil.CompareAndSetByPath` there, see this test: https://github.com/FerretDB/FerretDB/blob/630a7e9b336abaedb51ff6cd54b9219dbd4bee31/internal/handlers/handler_test.go#L554

Reply:
> And we never set it back.

Sure, I can reset the default value with `defer`.

> I think we should be fine with `testutil.CompareAndSetByPath` there, see this test

Could you, please, add more context here?
Should I remove this case into a separate test and use ` testutil.CompareAndSetByPath`?

Reply:
Fixed. Thanks!